### PR TITLE
Rewrite Started-ish

### PR DESCRIPTION
**Pull request by TsundereBug**
# Read this before you merge.

Config is located in `src/main/resources/config.properties`. Config loading should work fine. If not, build the jar and try again. If not even that works or you don't want to do it, you can edit the run configuration to pass in the location of the config in the command line arguments. No `--config` needed, just the path to the config.

Adding commands is done in `Commands`. If you have any questions about adding commands, please reply to this issue instead of having the questions lost in Discord.

Finally, this has not been tested.. There are `TODO` comments in `Command` and `Commands`. Please look at those and figure out how to show help. You can do it in `Command` by just reading the `getName() + "-help"` property in the language file.

Ping me on Discord if you have any questions about package or class naming in this new system, as it might be hard to follow.

**Let me repeat:** This has not been tested a single bit. Plugging in a token should work for now.

### CODE REVIEW

**Comment by TsundereBug** [comment]
Well, I started testing it, and there's still a few complex bugs. **Do not merge yet.** I am working on them now.

**Comment by TsundereBug** [comment]
<:rainblob:304759070680809474> Woo! Fixed everything! Should be good to add commands now. Prefix is kinda taken care of.

**Comment by TsundereBug** [comment]
Note that as of now commands are like `.horo ping` and not `.horoping`. I will fix this in this PR if you want me to, or if you want me to fix it later and you just want to add commands now.

**Comment by WinteryFox** [comment]
You should fix the prefix to be 1 thing with the command again, else if you change the prefix it'd be like % ping instead of just %ping. It's called a prefix for a reason :p

**Comment by WinteryFox** [comment]
Also we should document everything, literally everything. So we don't constantly have to ask what does what and why 'n other stuff

**Comment by WinteryFox** [comment]
you can do that later though, I'mma just merge cause I wanna do some stuff too
